### PR TITLE
Testing single producer

### PR DESCRIPTION
**Pull request by grzanka**
To test it on `lxplus` run:
```
source /cvmfs/cms.cern.ch/cmsset_default.sh
cmsrel CMSSW_12_3_X_2022-01-23-2300
cd CMSSW_12_3_X_2022-01-23-2300/src
cmsenv
git cms-checkout-topic CTPPS:pps_AlCaRecoProducer_single
scram b -j8
./Calibration/PPSAlCaRecoProducer/test/test_prompt_AlCaRecoProducer.sh
cmsRun Calibration/PPSAlCaRecoProducer/test/test_prompt_PPSAlCaReco_output.py
./Calibration/PPSAlCaRecoProducer/test/test_express_AlCaRecoProducer.sh
cmsRun Calibration/PPSAlCaRecoProducer/test/test_express_PPSAlCaReco_output.py
```

### CODE REVIEW

File: Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py [hunk 1]
```diff
@@ -7,28 +7,62 @@
   HLTPaths = ['HLT_PPSMaxTracksPerArm1_v1']
 )
 
-from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import totemTriggerRawToDigi as _totemTriggerRawToDigi
 from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import totemRPRawToDigi as _totemRPRawToDigi
 from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import ctppsDiamondRawToDigi as _ctppsDiamondRawToDigi
 from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import totemTimingRawToDigi as _totemTimingRawToDigi
 from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import ctppsPixelDigis as _ctppsPixelDigis
```
Comment (ChrisMisan) [inline Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py:13]:
```suggestion
from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import *
```

File: Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py [hunk 2]
```diff
@@ -7,28 +7,62 @@
...
+#ctppsDiamondRawToDigi  = _ctppsDiamondRawToDigi.clone(rawDataTag = 'hltPPSCalibrationRaw')
+#totemTimingRawToDigi = _totemTimingRawToDigi.clone(rawDataTag = 'hltPPSCalibrationRaw')
+ctppsPixelDigis = _ctppsPixelDigis.clone(inputLabel = 'hltPPSCalibrationRaw')
+
+ctppsRawToDigiTask = cms.Task(
+  #totemRPRawToDigi,
+  #ctppsDiamondRawToDigi,
+  # totemTimingRawToDigi,
+  ctppsPixelDigis
 )
```
Comment (ChrisMisan) [inline Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py:25]:
```suggestion
otemRPRawToDigi.rawDataTag = cms.InputTag('hltPPSCalibrationRaw')
ctppsDiamondRawToDigi.rawDataTag = cms.InputTag('hltPPSCalibrationRaw')
totemTimingRawToDigi.rawDataTag = cms.InputTag('hltPPSCalibrationRaw')
ctppsPixelDigis.inputLabel = cms.InputTag('hltPPSCalibrationRaw')
ctppsRawToDigiTask = cms.Task(
  totemRPRawToDigi,
  ctppsDiamondRawToDigi,
  totemTimingRawToDigi,
  ctppsPixelDigis
)
)
```

File: Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py [hunk 3]
```diff
@@ -7,28 +7,62 @@
...
+
+from RecoPPS.ProtonReconstruction.ctppsProtons_cff import *
+
+from Geometry.VeryForwardGeometry.geometryRPFromDB_cfi import *
+from CalibPPS.ESProducers.ppsTopology_cff import *
+
+recoCTPPSTask = cms.Task(
+    # totemRPLocalReconstructionTask ,
+    # ctppsDiamondLocalReconstructionTask ,
+    # diamondSampicLocalReconstructionTask ,
```
Comment (ChrisMisan) [inline Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py:60]:
```suggestion
    totemRPLocalReconstructionTask ,
    ctppsDiamondLocalReconstructionTask ,
    diamondSampicLocalReconstructionTask ,
```

**Comment by grzanka** [comment]
> I think you're missing some configuration, please check if above works for you.

Thanks, it really helped. I've added your changes in the last commit.
Can you make one more test?

I've tried to inspect the reconstructed objects and it seems that there are no tracks being produced.

To generate plots I've used:
```
cmsRun Calibration/PPSAlCaRecoProducer/test/test_express_PPSAlCaReco_output.py
```

**Comment by ChrisMisan** [comment]
I don't think the produced file is empty, I plugged the file into the diamond DQM and the plots seem fine. Maybe the issue lies in how you plot the results?
image
image

**Pull request merged**